client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -49,17 +127,24 @@ button.random-query--button {
 }
 
 
-button.random-query--button:hover,
-button.random-query--button:focus {
+button.random-query--button:hover {
   color: #003C32;
   border-color: #003C32;
 }
 
+button:focus {
```

Comment:
there are 2 other buttons -> "Retrieve Answers" and the "Show full answer"/"Collapse full answer". global styles are not the goal for modular components. if we want global styles, they should reside in the `App.css` file